ros2 launch passes extra --ros-args flag

Hello. When launching kumo bridge from a ros2 launch file, extra parameters are passed which cause parse_args() to error. 
I did some searching but couldn't figure out how to turn this off from the ros2 side of things.

    parallels@ubuntu:/media/psf/Workspace$ ros2 launch launch/demo.py 
    [INFO] [launch]: All log files can be found below /home/parallels/.ros/log/2023-04-06-10-50-16-794536-ubuntu-547275
    [INFO] [launch]: Default logging verbosity is set to INFO
    [INFO] [bridge-1]: process started with pid [547277]
    [bridge-1] usage: ros2 run kumo bridge [-h] [--port PORT] [--hosts HOSTS [HOSTS ...]]
    [bridge-1] ros2 run kumo bridge: error: unrecognized arguments: --ros-args
    [ERROR] [bridge-1]: process has died [pid 547277, exit code 2, cmd '/media/psf/Workspace/install/kumo/lib/kumo/bridge --ros-args'].

A one-line fix would be changing this line in scripts/bridge.py to just ignore the extra parameters:

    -    arg = parser.parse_args()
    +    (arg, _) = parser.parse_known_args()

I can open a PR.

--- a/scripts/bridge.py
+++ b/scripts/bridge.py
@@ -33,7 +33,7 @@ def main():
     parser.add_argument('--hosts', nargs='+',
                         help='specify host addresses to be bind by the bridge server')
 
-    arg = parser.parse_args()
+    (arg, _) = parser.parse_known_args()
 
     Bridge(arg.port, arg.hosts).run()
 

